src/App.tsx

```diff
@@ -1,12 +1,62 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import { useState } from "react";
+import { makeStyles } from "@mui/styles";
+import { Button } from "@mui/material";
+import { Box } from "@mui/system";
 
 
-function App() {
+const useStyles = makeStyles({
+  container: {
+    width: '90%',
+    margin: '50px auto',
+    textAlign: 'center'
+  },
+  h1: {
+    color: 'burlywood',
+    fontSize: '3rem',
+  },
+  board: {
+    maxWidth: '450px',
+    margin: '25px auto',
+    display: 'flex',
+    flexWrap: 'wrap',
+  },
+  square: {
```

Comment:
Let's change this to `cell` so it's more adequate to the context

Reply:
Aight!